book/analytics/client_side_api/client_side_api.md

```diff
@@ -126,6 +128,23 @@ Default properties are properties that you want to send along as an event proper
 
 The Morpheus Analytic Storage API enables Applicaster client developers to deliver user profile and default properties to the analytics storage container where this data is held.
 
+Please carefully read the following segment on PII vs Non PII data - and make sure to use the user properties function with the correct distribution of fields between general and personally identifiable information accordingly
+
+##### PII data vs. Non PII data
+
+Personally identifiable Information (PII) is information which can be used to distinguish or trace an individual's identity alone, such as their name, social security number, biometric records, etc., or can be used to identify an individual when combined with other personal or identifying information which is linked or linkable to the specific individual, such as date and place of birth, mother’s maiden name, etc.
+
+Applicaster currently supports several fields which are considered PII. Morpheus, our infrastructure allowes seperating these two types of fields to different groups when setting them so that they will not be sent to providers who do not accept PII. This also enables customers to filter out the delivery of PII because of regional/legal needs even to providers who can receive PII. Please make sure to map any PII data and send it only in the PII dictionary / hash accordingly.
```

Comment:
```suggestion
Applicaster currently supports several fields which are considered PII. Morpheus, our infrastructure allows separating these two types of fields to different groups when setting them so that they will not be sent to providers who do not accept PII. This also enables customers to filter out the delivery of PII because of regional/legal needs even to providers who can receive PII. Please make sure to map any PII data and send it only in the PII dictionary / hash accordingly.
```

Reply:
May want to make Morpheus link to the TechBook article on it